bin/helm_toolbox.sh

```diff
@@ -17,40 +19,33 @@ function check() {
 	command -v "$1" >/dev/null 2>&1 || err "$1 not installed!"; 
 }
 
-# wait for helm to become operational
-wait_for_helm() {
-	info "Waiting for helm tiller..."
-	while true; do
-		status=$(echo $(kubectl get pods -l app=helm -l name=tiller --show-all=false -o=custom-columns=STATUS:.status.phase --no-headers=true -nkube-system))
-		info "Helm status: $status"
-		if [ "$status" = "Running" ]; then
-			break;
-		fi
-		sleep $TIMEOUT
-	done
-}
-
 function set_context() {
   if [ -n "$KUBE_CONTEXT" ]; then
 	info "Using ${KUBE_CONTEXT} kube context"
 	kubectl config use-context ${KUBE_CONTEXT}
   fi
 }
 
+function retry() {
+  local attempt=0
+	local limit=$1
+	local command=${@:2}
+  until [[ $attempt -ge $limit ]]
+  do
+		info "Perform attempt - $[$attempt+1]"
```

Comment:
Indent looks off on this line and the one below. What could help with this in the future is adding a `*.sh` config here: https://github.com/cloudposse/build-harness/blob/master/.editorconfig and ensuring you have the `editorconfig` plugin installed for your editor.

Here is an example of sh editorconfig we use https://github.com/cloudposse/terraform-root-modules/blob/master/.editorconfig#L22-L24

Reply:
Yea, make sure your editor supports `.editorconfig`

https://docs.cloudposse.com/development/editor-config/